action.yml

```
@@ -262,6 +277,9 @@ runs:
           core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env['ACTIONS_RUNTIME_TOKEN'])
           core.exportVariable('ACTIONS_RUNTIME_URL', process.env['ACTIONS_RUNTIME_URL'])
 
+    - name: create cache dir
+      run: |
+        mkdir -p $GITHUB_WORKSPACE/cache
```

Need to also add shell: bash because this is failing too
```suggestion
    - name: create cache dir
      run: |
        mkdir -p $GITHUB_WORKSPACE/cache
      shell: bash
```